index.js

```diff
@@ -64,6 +65,13 @@ const init = async () => {
     delete require.cache[require.resolve(`./events/${file}`)];
   });
 
+  const backgroundFiles = await readdir("./background/");
+  client.logger.log(`Loading a total of ${backgroundFiles.length} background workers.`);
+  backgroundFiles.forEach(file => {
```

Comment:
This would be much better if you did

```javascript
await Promise.all(backgroundFiles.map(file => {
    const workerName = file.split(".")[0];
    return client.loadBackgroundWorker(workerName);
}));
```